CustomComponent.md

```diff
@@ -0,0 +1,36 @@
+# Sharing Infrastructure Patterns Across an Organization's Development Teams
+
+No matter what role I have had or where I have worked, one thing that has stuck
```

Comment:
The intro into the topic needs to capture more about how organizations use common languages and frameworks across many development teams and how sharing infrastructure patterns across teams can make it easier for developers to focus on application requirements.